src/github.com/matrix-org/dendrite/publicroomsapi/consumers/roomserver.go

```diff
@@ -39,13 +39,13 @@ func NewOutputRoomEvent(
 	kafkaConsumer sarama.Consumer,
 	store *storage.PublicRoomsServerDatabase,
 	queryAPI api.RoomserverQueryAPI,
-) *OutputRoomEvent {
+) *OutputRoomEventConsumer {
```

Comment:
github won't let me put this on the right line, but this NewOutputRoomEvent needs a rename too